packages/patternfly-4/src/pages/index.js

```diff
@@ -1,20 +1,215 @@
+      <Grid>
+        <GridItem sm={12} md={4} mdOffset={4} className="pf-u-text-align-center">
+          <Title size="4xl" id="AboutPatternFly4" className="pf-u-mb-md">About PatternFly 4</Title>
+          <Title size="lg">We provide tools like design documentation, components, and code examples to make it possible for anyone to design and build responsive, accessible user experiences.</Title>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection variant={PageSectionVariants.light} className="pf4-c-background-lines">
+      <Grid gutter="md" className="pf-u-pb-3xl">
+        <GridItem sm={12}>
+          <Grid>
+            <GridItem sm={12} md={4} mdOffset={2}>
+              <TextContent>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Make better decisions with clear guidance</Text>
+                  <Text component={TextVariants.p}>Design guidelines supply the foundation of the system. Get acquainted with our approach to icons, color, content, and more.</Text>
+                  <Text component={TextVariants.a} href="/design-guidelines/styles/icons" aria-label="view design guidelines" className="pf-u-pl-0 pf4-c-link__text">View design guidelines <CaretRightIcon />
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Start building with components</Text>
+                  <Text component={TextVariants.p}>Components are like building blocks. Designed to be flexible and modular, you can mix and match to create a solution for almost any UI problem.</Text>
+                  <Text component={TextVariants.a} href="/documentation/react/components/alert" aria-label="view components" className="pf-u-pl-0 pf4-c-link__text">View components <CaretRightIcon />
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Stay aligned with layouts</Text>
+                  <Text component={TextVariants.p}>Layouts provide fully responsive page structure that keeps your components organized and aligned regardless of screen size.</Text>
+                  <Text component={TextVariants.a} href="/documentation/react/layouts/bullseye" aria-label="view layouts" className="pf-u-pl-0 pf4-c-link__text">View layouts <CaretRightIcon />
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Get inspired with demos</Text>
+                  <Text component={TextVariants.p}>Explore working examples of common UI elements like forms to see how components and layouts can be combined to solve common design problems.</Text>
+                  <Text component={TextVariants.a} href="/documentation/react/demos/pagelayout" aria-label="view demos" className="pf-u-pl-0 pf4-c-link__text">View demos <CaretRightIcon />
```

Comment:
href to `/documentation` should work

Reply:
With each one supposed to go to a different location, should I only change the first reference to documentation?

Reply:
Oh, if they are to specific demos then yes you can leave the URL. Just in case where you want to generally guide them towards the Demos page then you can use a shorter href